test/files/pos/t7232e/Foo.java

```diff
@@ -0,0 +1,10 @@
+package pack;
+
+import java.util.List;   // does shadow
+import java.awt.*;       // doesn't shadow, keep looking
+import java.util.function.Supplier;  // irrelevant, keep looking
```

Comment:
The irrelevant import serves the same role as the other test. However, the `awt` wildcard also does not shadow in Java, so don't pick it during lookup.

This test was the only time it was ever useful to me to have `java.awt.List`, which for years messed with autocomplete.